src/nnet3/nnet-batch-compute.h

```diff
@@ -193,6 +193,9 @@ struct NnetBatchComputerOptions: public NnetSimpleComputationOptions {
 void MergeTaskOutput(
     const std::vector<NnetInferenceTask> &tasks,
     Matrix<BaseFloat> *output);
+void MergeTaskOutputGPU(
```

Comment:
Thanks.  You can just call it MergeTaskOutput() and let the function overloading take care of it, that's what I normally do for CPU/GPU things.  Incidentically, if it had the  suffix, it would be MergeTaskOutputGpu()... it's a Google style guide thing.

Reply:
Fixed